src/Renci.SshNet/IPrivateKeySource.cs

```diff
@@ -1,15 +1,22 @@
-﻿using Renci.SshNet.Security;
+﻿using System.ComponentModel;
+
+using Renci.SshNet.Security;
 
 namespace Renci.SshNet
 {
     /// <summary>
     /// Represents private key source interface.
     /// </summary>
-    public interface IPrivateKeySource
+    /// <remarks>
+    /// This interface has been replaced by <see cref="IHostAlgorithmsProvider"/>
+    /// and is currently not used in the library.
+    /// </remarks>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Maybe we should add [Obsolete("comment")]?

Reply:
Done. I initially could not decide but agree it is the better thing to do